### PR TITLE
Ops/deployment scripts

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/ExporterBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
 
   transformcore.phe.exporter.compression:
     class: TransformCore\PHE\ExporterBundle\Service\FileCompressionService
-    arguments: ['%export_path%', 'archived/']
+    arguments: ['%export_path%', 'archived']
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Resources/config/services.yml]: Changed back?

**Comment by eddiejaoude** [comment]
`make` might be better than shell scripts as can reuse the building blocks - just a thought for the future :)

File: src/TransformCore/PHE/ExporterBundle/Resources/config/services.yml (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Resources/config/services.yml]: Yeah, I was adding the / twice, the code itself uses DIR_SEP so is more robust

**Comment by brettminnie** [comment]
Noted for future

**Pull request merged**